### PR TITLE
Python: fix `mkdocs` warnings from `sorting.py`

### PR DESCRIPTION
**Pull request by DarthData410**
After review with @Fokko the **data (Keyword Args) is where the packed argument representing order_id is expected. Changed the structure of comments to match this fact and mkdocs works fine. Ref: https://mkdocstrings.github.io/griffe/docstrings/#other-parameters issue #7458

### CODE REVIEW

File: python/pyiceberg/table/sorting.py [hunk 1]
```diff
@@ -115,10 +115,12 @@ class SortOrder(IcebergBaseModel):
...
 
     Args:
+      fields (List[SortField]): The fields how the table is sorted.
+
+    Keyword Args:
       order_id (int): An unique id of the sort-order of a table.
-      fields (List[SortField]): The fields how the table is sorted
     """
-
+
```
Comment (Fokko) [inline python/pyiceberg/table/sorting.py:123]: Can you remove this newline?

Comment (DarthData410) [reply python/pyiceberg/table/sorting.py]: per request, has bee removed

Comment (Fokko) [reply python/pyiceberg/table/sorting.py]: Looks like the linter doesn't agree. Can you add the newline, and remove the spaces. I misread the change.

Comment (DarthData410) [reply python/pyiceberg/table/sorting.py:123]: ... done...

**Comment by DarthData410** [comment]
I'm about to redo this real quick from scratch...

**Comment by DarthData410** [comment]
never mind... I'm not sure why this was not being removed... I'm using vscode, and it has worked fine. Sorry for the trouble on this, will double check in the future to not cause issues.

**Comment by Fokko** [comment]
@DarthData410 no worries, for some reason, VSCode adds spaces when it isn't needed.

**Comment by Fokko** [comment]
Thanks again @DarthData410

**Pull request merged**